v4.13/catalog/rhtas-operator/catalog.json

```diff
@@ -9,26 +9,30 @@
     "package": "rhtas-operator",
     "entries": [
         {
-            "name": "rhtas-operator.v1.0.0"
+            "name": "rhtas-operator.v1.0.0",
+            "message": "This version is deprecated. Please upgrade to rhtas-operator.v1.1.0."
```

Comment:
Please correct me if I am wrong, but I believe you are making these changes manually ?, The catalog.json file is something that is generated using the opm tool, you can find more info [here](https://pages.rhtas.com/technical-guides/release-coordination/#generating-a-new-build-of-the-operator) in the FBC bits

Reply:
Haha yes, good point :sweat_smile: